ddtrace/_trace/utils_botocore/aws_payload_tagging.py

```diff
@@ -0,0 +1,242 @@
+import copy
+from decimal import Decimal
+import json
+from typing import Any
+from typing import Dict
+from typing import Optional
+
+from ddtrace import Span
+from ddtrace import config
+from ddtrace.vendor.jsonpath_ng import parse
+
+
+_MAX_TAG_VALUE_LENGTH = 5000
+
+
+class AWSPayloadTagging:
```

Comment:
Many methods of this class seem to ignore the `self` argument. This is maybe an indication that things could be written differently?

Reply:
@P403n1x87 Maybe? I'm honestly unsure and would need help to determine what a better solution would look like.